response/slack/cache.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from django.conf import settings
+from django.db import transaction
+
+from response.core.models import ExternalUser, GetOrCreateSlackExternalUser
+from response.slack.client import SlackError
+
+logger = logging.getLogger(__name__)
+
+
+def update_user_cache():
+    cursor = None
+    while True:
```

Comment:
You could do `while cursor != "":` here and remove the check below (I've just remembered that https://github.com/monzo/response/pull/85 does this)